### PR TITLE
(SIMP-7826) Add support for KeepAlive variables

### PR DESCRIPTION
**Pull request by kendall-moore**
* Add support to both the imtcp module and the built-in omfwd module for
the KeepAlive variables that were added in rsyslog v8

SIMP-7826 #close

### CODE REVIEW

**Comment by kendall-moore** [comment]
Reference CI pipeline: https://gitlab.com/simp/pupmod-simp-rsyslog/-/pipelines/158821255

File: templates/config/pre_logging.conf.erb [hunk 1]
```diff
@@ -126,6 +126,15 @@ module(load="imtcp"
   StreamDriver.AuthMode="<%= @_action_send_stream_driver_auth_mode %>"
   PermittedPeer=[<%= @tls_input_tcp_server_stream_driver_permitted_peers.map{|x| %Q{"#{x}"}}.join(',') %>]
   MaxSessions="<%= @tls_tcp_max_sessions %>"
+<%   if @keep_alive -%>
```
Comment (silug) [inline templates/config/pre_logging.conf.erb:129]: Should this be `unless @keep_alive.nil?` maybe?  Won't `0` look falsey?

Comment (silug) [reply templates/config/pre_logging.conf.erb:129]: > Should this be `unless @keep_alive.nil?` maybe? Won't `0` look falsey?

To answer my own question, no.

**Comment by trevor-vaughan** [comment]
Running this through the `simp-core` tests now

**Comment by trevor-vaughan** [comment]
Looks like everything is good to go.

**Pull request merged**